ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -48,48 +39,40 @@ export class ImportCommand extends IronfishCommand {
 
     const client = await this.sdk.connectRpc()
 
-    let account: RpcAccountImport
+    let account: string
     if (blob) {
-      account = await this.stringToRpcAccountImport(blob, flags)
+      account = blob
     } else if (flags.path) {
-      account = await this.importFile(flags.path, flags)
+      account = await this.importFile(flags.path)
     } else if (process.stdin.isTTY) {
-      account = await this.importTTY(flags)
+      account = await this.importTTY()
     } else if (!process.stdin.isTTY) {
-      account = await this.importPipe(flags)
+      account = await this.importPipe()
     } else {
       CliUx.ux.error(`Invalid import type`)
     }
 
-    if (!account.version) {
-      account.version = ACCOUNT_SCHEMA_VERSION
-    }
-
-    if (!account.createdAt) {
-      account.createdAt = null
-    }
-
     const accountsResponse = await client.wallet.getAccounts()
     const duplicateAccount = accountsResponse.content.accounts.find(
-      (accountName) => accountName === account.name,
+      (accountName) => accountName === flags.name,
```

Comment:
Since we don't parse the account on the client side anymore we won't be able to prompt users to choose a different name if the encoded account has a duplicate name

That's probably fine, but maybe we don't need to have the prompt at all if it can't cover that case?

Reply:
By client side you mean in the CLI? We still can check for duplicate names for mnemonic/spending key, but you are right about it not being possible for the other types (bech32/json)

Reply:
Addressed this here:
https://github.com/iron-fish/ironfish/pull/4035